Remove the hardcoded lib and include dirs

Upstream-Status: Inappropriate [cross compile specific]

written by: Amy Fong <amy.fong@windriver.com>
Signed-off-by: Jackie Huang <jackie.huang@windriver.com>

--- keynote-2.3/configure.in.orig	2010-05-24 04:44:16.000000000 -0700
+++ keynote-2.3/configure.in	2010-05-24 04:44:55.000000000 -0700
@@ -21,27 +21,16 @@
 AC_PATH_PROG(ECHO, echo, /bin/echo)
 AC_PATH_PROG(SED, sed, /usr/bin/sed)
 
-dnl Checks for libraries.
-LIBS="-L/usr/lib -L/usr/local/lib -L/usr/ssl/lib -L/usr/openssl/lib\
- -L/usr/local/ssl/lib -L/usr/local/openssl/lib -L/usr/pkg/lib -L/pkg/lib"
-
 AC_CHECK_LIB(m, floor, LIBS="$LIBS -lm")
 AC_CHECK_LIB(rsaref, RSAPrivateDecrypt, LIBS="$LIBS -lrsaref")
 AC_CHECK_LIB(crypto, i2a_ASN1_STRING, LIBS="$LIBS -lcrypto")
 AC_CHECK_LIB(RSAglue, RSA_ref_private_encrypt, LIBS="$LIBS -lRSAglue")
 
-dnl Checks for header files.
-CPPFLAGS="-I/usr/include -I/usr/local/include -I/usr/ssl/include\
- -I/usr/local/ssl/include -I/usr/openssl/include -I/usr/pkg/include\
- -I/usr/local/openssl/include -I/pkg/include"
-
 AC_HEADER_STDC
 AC_HEADER_TIME
 AC_CHECK_HEADERS(fcntl.h limits.h unistd.h regex.h sys/time.h io.h)
 AC_CHECK_HEADERS(ssl/crypto.h openssl/crypto.h crypto.h memory.h)
 
-dnl Checks for other files
-
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
 AC_CHECK_TYPE(u_int, unsigned int)
